shared/controller-api/src/main/java/io/pravega/shared/controller/event/ScaleOpEvent.java

```diff
@@ -24,9 +24,10 @@
     private final List<Long> segmentsToSeal;
     private final List<AbstractMap.SimpleEntry<Long, Long>> newRanges;
     private final boolean runOnlyIfStarted;
-    private final long scaleTime;
+    private final long scaleTime; // scaleTime is an equivalent for requestId
```

Comment:
Same here about overloading fields.

Reply:
Now `ScaleOpEvent` and `AutoScaleEvent` contain a `requestId` field for tracing.